libcontainer/rootfs_linux.go

```diff
@@ -1089,7 +1089,21 @@ func mountPropagate(m *configs.Mount, rootfs string, mountLabel string, mountFd
 	if err := utils.WithProcfd(rootfs, m.Destination, func(procfd string) error {
 		return mount(source, m.Destination, procfd, m.Device, uintptr(flags), data)
 	}); err != nil {
-		return err
+		if m.Device == "sysfs" {
+			/* If we are running in a user namespace, just bind mount /sys if creating
+			   sysfs failed.  */
```

Comment:
Any reason not to let the higher level runtime generate config.json with this bind mount ?

Reply:
This is problematic because /sys/class/net entries are network-namespaced, if you use privileged mode, there may be escape problems, but rootless don't need to worry